lib/srv/desktop/rdp/rdpclient/src/vchan.rs

```diff
@@ -18,6 +18,8 @@ use bitflags::bitflags;
 use byteorder::{LittleEndian, ReadBytesExt, WriteBytesExt};
 use rdp::model::error::*;
 
+pub const MAX_CHUNK_SIZE: usize = 1600;
```

Comment:
Note to reviewers: technically the RDP server can support reading larger chunks than this, and it sends its maximum chunk size as a capability during connection establishment.

1600 bytes is the default, and since we only use this on send when breaking up large messages, we'll still be in compliance even if we're breaking a message up into smaller chunks than necessary.

Reply:
I propose we add a comment here explaining what this is.

Reply:
Sounds good. I'll link to https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-rdpbcgr/6c074267-1b32-4ceb-9496-2eb941a23e6b